js/node.js

```diff
@@ -0,0 +1,212 @@
+                        break;
+
+                        case "last_updated":
+                            elem.append((converters[key] || converters["last_updated"])(val));
+                        break;
+
+                        case "contact":
+                            let contact_email = document.getElementById("contact");
+                            elem.append((converters[key] || converters["_default"])(val));
+                        break;
+
+                        case "version":
+                            let node_version = document.getElementById("version");
+                            node_version.textContent = val;
+                            elem.append((converters[key] || converters["_default"])(val));
+                        break;
+
+                        case "services":
+                            elem.append((converters[key] || converters["services"])(val));
+                        break;
+
+                        case "url":
+                            elem.append((converters[key] || converters["_default"])(val));
+                        default:
+
+                            elem.append((converters[key] || converters["_default"])(val));
+                    }
+
+                }
+            })
+        }
+
+        node_info.links.forEach(link => {
+            if (link.rel === "service") {
+                const elem = document.getElementById("url");
+                const link_elem = document.createElement("a");
+                let small_logo_link = document.getElementById("small-logo-link");
+                small_logo_link.href = link.href;
+
+                Object.entries(link).forEach(([attr, value]) => link_elem.setAttribute(attr, value));
+                link_elem.innerText = link.href;
+                elem.appendChild(link_elem)
+            } else if (link.rel === "icon") {
+                const icon_img = document.createElement("img")
+                icon_img.setAttribute("src", link.href)
+
+                const background_elem = document.getElementById("background-icon");
+                background_elem.style.backgroundImage="url(" + link.href + ")";
+                background_elem.style.backgroundRepeat = "no-repeat";
+                background_elem.style.backgroundSize = "cover";
+                background_elem.style.width = "100%";
+                background_elem.style.height = "100%";
+                background_elem.classList.add("info-background");
+
+                const title_icon_img = document.getElementById("title-icon-img")
+                title_icon_img.classList.add("img-fluid", "d-block", "float-end");
+                title_icon_img.setAttribute("src", link.href);
+
+                //Used to position the sharp logo overlayed on the blurred logo.
+                //This effect may be used in the future
+                /*
+                switch(node_name) {
```

Comment:
I know you commented this out in case we want to add this feature back in the future but the code that is commented uses the hard-coded node names and so cannot be implemented as-is even if we wanted to. Please just remove this comment.

Reply:
Comment removed